src/Compilers/CSharp/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1,8 +1,15 @@
+*REMOVED*Microsoft.CodeAnalysis.CSharp.Syntax.ArgumentSyntax.Update(Microsoft.CodeAnalysis.CSharp.Syntax.NameColonSyntax nameColon, Microsoft.CodeAnalysis.SyntaxToken refOrOutKeyword, Microsoft.CodeAnalysis.CSharp.Syntax.ExpressionSyntax expression) -> Microsoft.CodeAnalysis.CSharp.Syntax.ArgumentSyntax
```

Comment:
Technically a (source) breaking change here. Consulting the Compatibility Council.

Reply:
Yes, this was intended and reviewed.